yuniql-cli/Program.cs

```diff
@@ -74,6 +74,8 @@ private static int Dispatch<T>(Func<T, int> command, T opts, ITraceService trace
             Console.ResetColor();
 
             traceService.IsDebugEnabled = opts.IsDebug;
+            traceService.TraceSensitiveData = opts.TraceSensitiveData;
```

Comment:
As convention, we start all boolena properties with `Is` :)

Reply:
thanks for the feedback! :) fixed with latest commit